### PR TITLE
Content Cards customizability update documentation related to release

### PR DESCRIPTION
**Pull request by tbelote**
# Pull Request/Issue Resolution
SDK-1269

**Description of Change:**
The iOS SDK for Content Cards is changing to allow for greater flexibility and customization.

### Is this change associated with a Braze feature/product release?
- [X] Yes Appboy iOS SDK 4.2.0 exact date TBD

---
---

<details>
<summary>PR Checklist</summary>

- [ ] Check that all links work!
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is _done and ready to be reviewed for merge_.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/ios/content_cards/customization.md [hunk 1]
```diff
@@ -96,4 +96,4 @@ Note that overriding default images is currently not supported in our Xamarin iO
...
+You can create your own Content Cards interface by extending `ABKContentCardsTableViewController` to customize all UI elements and Content Cards behavior. The ContentCard cells may also be subclassed in particular the `setUpUI` method is powerful for customization and then registered with the tableview either by implementing `registerTableViewCellClasses` or using a custom Storyboard that registers your new classes. See the [Content Cards sample app](https://github.com/Appboy/appboy-ios-sdk/tree/master/Samples/ContentCards/BrazeContentCardsSampleApp) for a more complete example. Alternatively, you can create a completely custom view controller and [subscribe for data updates]({{site.baseurl}}/developer_guide/platform_integration_guides/ios/content_cards/data_model/). In the latter case, you would need to log all view events, dismissed events, and clicks manually.
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/ios/content_cards/customization.md:99]:
```suggestion
You can create your own Content Cards interface by extending `ABKContentCardsTableViewController` to customize all UI elements and Content Cards behavior. The Content Card cells may also be subclassed and then used programmatically or by introducing a custom Storyboard that registers the new classes. See the [Content Cards sample app](https://github.com/Appboy/appboy-ios-sdk/tree/master/Samples/ContentCards/BrazeContentCardsSampleApp) for a more complete example. Alternatively, you can create a completely custom view controller and [subscribe for data updates]({{site.baseurl}}/developer_guide/platform_integration_guides/ios/content_cards/data_model/). In the latter case, you would need to log all view events, dismissed events, and clicks manually.
```

**Review by hokstuff — DISMISSED** [review]
One suggestion

**Review by Bucimis — APPROVED** [review]
lgtm after changes, reminder that this should be merged only after release

**Review by chshapiro — DISMISSED** [review]

**Comment by KellieHawks** [comment]
@tbelote Just checking in, is this PR good to merge?

**Comment by hokstuff** [comment]
Not yet, we actually are planning to release these docs changes with `4.2.0` which we aim to release next week. I'll update the PR description

**Comment by Bucimis** [comment]
@KellieHawks this is ready to merge (@tbelote sanity)

**Review by KellieHawks — APPROVED** [review]

**Pull request merged**